common/beerocks/bwl/nl80211/ap_wlan_hal_nl80211.cpp

```diff
@@ -13,6 +13,7 @@
 #include <bcl/beerocks_version.h>
 #include <bcl/network/network_utils.h>
 #include <bcl/son/son_wireless_utils.h>
+#include <configuration.h>
```

Comment:
I didn't notice this in the other PR - the header files should be in a subdirectory that indicates the module, i.e. it should be in common/beerocks/prplmesh_hostapd/include/prplmesh_hostapd/configuration.h and we should put here
```suggestion
#include <prplmesh_hostapd/configuration.h>
```

Reply:
oh.. crap... just now solved it... this is so unnatural that I really had to dig into it till I figured it out:
`./common/beerocks/bwl/include/bwl/` <---- **bwl**/include/**bwl**
`./common/beerocks/bcl/include/bcl/` <---- **bcl**/include/**bcl**
now I added:
`./common/beerocks/hostapd/include/hostapd/` to follow current scheme.

I am not dealing with fixing it right now, the issue supposed to be refactored in [PPM-219](https://jira.prplfoundation.org/browse/PPM-219)